doc/discretization.rst

```diff
@@ -0,0 +1,282 @@
+=======================
+Discretization Strategy
+=======================
+
+.. _disc-strat:
+
+Literature on DG for Fluid Flow
+===============================
+
+How to discretize the conservation equations with DG, including how to handle the required
+fluxes, particularly in the viscous setting, is a current topic of research and internal
+discussion.  The following references are useful:
+
+* "The DG Book:" Nodal Discontinuous Galerkin Methods, [Hesthaven_2008]_
+* The BR1 method for discretization of Navier-Stokes, [Bassi_1997]_
+* NS with reactions, [Ihme_2014]_, and [Cook_2009]_
+* The BR2 method, [Bassi_2000]_
+* [Ayuso_2009]_
+
+*MIRGE-Com* currently employs a strategy akin to the BR1 algorithm outlined in
+[Bassi_1997]_, but with thermal terms and chemical reaction sources as outlined in
+[Ihme_2014]_ and [Cook_2009]_.
+
+Nodal DG for Navier-Stokes
+==========================
+
+The main system of equations we solve in the BR1 approach is summarized as follows:
+
+The compressible NS equations are rewritten as the following coupled system for two
+unknowns, $\mathbf{Q}$ and $\mathbf{\Sigma}$:
+
+$$
+\mathbf{\Sigma} - \nabla{\mathbf{Q}} &= \mathbf{0}\quad \text{aux eqn}\\
+\frac{\partial \mathbf{Q}}{\partial t} + \underbrace{\nabla\cdot\mathbf{F}^I(\mathbf{Q}) -
+\nabla\cdot\mathbf{F}^V(\mathbf{Q},\mathbf{\Sigma})}_{= \nabla\cdot\mathbf{F}(\mathbf{Q},
+\mathbf{\Sigma})} &= \mathbf{S} \quad \text{primary eqn}
```

Comment:
```suggestion
\mathbf{\Sigma})} &= \mathbf{S} \tag{\text{primary eqn}}
```

Reply:
Backed out and unresolved due to latex CI hating these.